src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -526,7 +477,7 @@ def convert_from_pytorch(pt_state: Dict, config: BertConfig) -> Dict:
                 jax_state[key] = tensor
 
             # There are some transposed parameters w.r.t their PyTorch counterpart
-            if "intermediate.dense.kernel" in key or "output.dense.kernel" in key:
+            if "intermediate.dense.kernel" in key or "output.dense.kernel" in key or "transform.dense.kernel" in key:
```

Comment:
To make BartForMaskedLM pass the integration tests.

Reply:
Isn't it just the weights that have `"kernel"` as their last part? That test is starting to get ugly.

Reply:
It's not all `"kernel"` weights that have that sadly. It's true that this test is getting ugly and I think the next PR should solve the way we convert PT to Flax models